payments-core/src/main/java/com/stripe/android/EphemeralKeyManager.kt

```diff
@@ -166,3 +166,5 @@ internal class EphemeralKeyManager(
         private const val REFRESH_BUFFER_IN_SECONDS = 30L
     }
 }
+
+internal typealias TimeSupplier = () -> Long
```

Comment:
This was previously defined in CustomerSession, but also used in this class. So I redefined it where it's actually being used.